docs/conf.py

```diff
@@ -84,6 +85,10 @@ def setup(app):
 # GitHub repo
 issues_github_path = 'scikit-hep/pyhf'
 
+# c.f. https://github.com/scikit-hep/scikit-hep.github.io/issues/278
+plausible_domain = "pyhf.readthedocs.io"
+plausible_script = "https://views.scientific-python.org/js/plausible.js"
```

Comment:
The [docs mention](https://github.com/rkdarst/sphinx-plausible/blob/e869fe4a27533189170963fcf2d89acddc066d3b/README.md?plain=1#L14-L25)

> ## Plausible without this extension
>
>Actually, this extension isn't needed, you can directly add a script
>with the required tags by doing the following.  This uses the same
>insertion mechanism as this extension, and thus should work with any
>well-designed theme with no modifications:
>
>```python
>html_js_files = [
>    ('https://plausible.io/js/script.js', {"data-domain": "yourdomain.com", "defer": "defer"}),
>]
>```

so if this would be perferable @kratsg I'm also willing to try this.